src/alfasim_sdk/_internal/alfacase/case_description.py

```diff
@@ -2829,7 +2962,8 @@ class PvtModelsDescription:
     correlations = attrib_dict_of(PvtModelCorrelationDescription)
     compositional = attrib_dict_of(PvtModelCompositionalDescription)
     combined = attrib_dict_of(PvtModelCombinedDescription)
-    table_parameters = attrib_dict_of(PvtModelTableParametersDescription)
```

Comment:
Renaming an existing property will cause alfacase files using the old key to fail the schema validation.
As of today we don't have any backward compatibility support for alfacase files.

Reply:
What can we do about it? Is there a way to avoid this problem?

Reply:
By "Looks like this was never present in alfacase files."
I was trying to convey that further investigation I think this change will have no actual impact becauseit never was present in alfacase.

Reply:
Also `IGNORED_PROPERTIES` in `src/alfasim_sdk/_internal/alfacase/generate_schema.py:274` must be updated.
Remove `table_parameters` and add the now 2 equivalent properties.

Reply:
I spent a long time yesterday figuring out why the `schema.py` have been updating when I call `inv cog`, and there is the answer. Thank you @prusse-martin!

Reply:
> Also `IGNORED_PROPERTIES` in `src/alfasim_sdk/_internal/alfacase/generate_schema.py:274` must be updated. Remove `table_parameters` and add the now 2 equivalent properties.

Thanks for this information. I didn't remember this detail.

Reply:
> By "Looks like this was never present in alfacase files." I was trying to convey that further investigation I think this change will have no actual impact becauseit never was present in alfacase.

You are right. Since this property is ignored, it was never present in the `alfacase`.